tests/core/conftest.py

```diff
@@ -219,3 +225,18 @@ async def form_bot_agent(trained_async) -> Agent:
     )
 
     return Agent.load_local_model(zipped_model)
+
+
+@pytest.fixture
+async def response_selector_agent(trained_async) -> Agent:
```

Comment:
missing type annotation
```suggestion
async def response_selector_agent(trained_async: Callable) -> Agent:
```